inspire/base/static/js/jquery.menu-aim.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * menu-aim is a jQuery plugin for dropdown menus that can differentiate
+ * between a user trying hover over a dropdown item vs trying to navigate into
```

Comment:
and then to add it as a dependency we do: 
bower install jQuery-menu-aim --save
?